app/views/Project/Tagging/Sources/SourcesTable/EntryList/EntryItem/EntryVerification/EntryUnverifyCommentModal/index.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useCallback } from 'react';
+import { _cs, isDefined } from '@togglecorp/fujs';
+import {
+    requiredStringCondition,
+    requiredListCondition,
+    ObjectSchema,
+    getErrorObject,
+    useForm,
+    requiredCondition,
+} from '@togglecorp/toggle-form';
+import {
+    Modal,
+    TextArea,
+    MultiSelectInput,
+    Button,
+} from '@the-deep/deep-ui';
+import { useRequest, useLazyRequest } from '#base/utils/restRequest';
+import NonFieldError from '#components/NonFieldError';
+import { EntryReviewComment } from '#types/newEntry';
+import {
+    MultiResponse,
+    Membership,
+} from '#types';
+
+import styles from './styles.css';
+
+const memberFieldQuery = {
+    fields: ['member', 'member_name'],
+};
+const UNVERIFY = 2;
```

Comment:
Lets keep all the enums in a util file or something similar and fetch it from there.